Update README.md

joshuabromero opened the pull request:
Adds section about how to access the admin panel and what the default username/password is.

joshuabromero committed:
Update README.md

Adds section about how to access the admin panel and what the default username/password is.

u-maxx commented on README.md:
```diff
@@ -192,6 +192,10 @@ The `-h` flag above (shorthand of `--help`) defines that setup script must only
 
+### Admin Panel
+The admin panel URL is auto-generated by the setup script. You can find the URL in the output in your terminal. The default username is `john.smith` and the default password is `password123`. You are free to change these default values in the following file:
+`compose/magento-2/bin/setup/start`
```
I do not think anyone needs to edit any compose/magento-2/bin/setup/* files manually before the install since our main focus is to let everyone just use automated setup command with the interactive or one-line mode. I think we can create additional parameters for one-line mode to be able to specify the default admin  url / user / password during the setup w/o editing any files

Could you please remove the following sentence from your changes : "You are free to change these default values in the following file:
`compose/magento-2/bin/setup/start`" ?

u-maxx replied:
I will create new GH issue to track this idea about additional parameters fro one-line / interactive setup

joshuabromero replied:
Sounds good. I'll update the changes.

shkoliar replied:
Additionally, any changes in bin/* scripts may be lost with `bin/update` or `bin/update -a` execution, because all scripts will be overwritten with newer versions.

joshuabromero replied:
That's a good point. Mumble, mumble, don't edit core. 😄

u-maxx replied:
well, they are allowed if you already have project files generated once via interactive setup for example, and keeping generated files in separate git repo... but you do not need to edit anything before installing/configuring project first time via interactive/one-line setup 100%

shkoliar commented on README.md:
```diff
@@ -192,6 +192,10 @@ The `-h` flag above (shorthand of `--help`) defines that setup script must only
 ## Misc Info
 
+### Admin Panel
+The admin panel URL is auto-generated by the setup script. You can find the URL in the output in your terminal. The default username is `john.smith` and the default password is `password123`. You are free to change these default values in the following file:
```
@magedevjosh, thanks for PR!

Would be good to add here command `bin/magento info:admin` which prints admin URL instead of searching admin URL in installation log.

joshuabromero replied:
I always forget about that command. Good catch. Updated.

joshuabromero committed:
Update README.md

Adds bin command to get admin url and removes sentence about changing default username/password.

u-maxx reviewed (approved):
Thanks @magedevjosh !
